python/examples/minimal_build/Dockerfile.fedora

```diff
@@ -27,6 +27,7 @@ RUN dnf update -y && \
         make \
         cmake \
         ninja-build \
+        pkg-config \
```

Comment:
why is this necessary?

Reply:
This is currently needed to install headers from the Arrow project into the PyArrow package. In theory, if CMake exposed a variable for the `ARROW_INCLUDE_DIR` then Meson could wrap that, but AFAIU in the current state that variable is not exposed by the CMake configuration, and is instead just pulled from the `TARGET_INCLUDE_DIRECTORIES` property of the arrow target.

See also the comment in pyarrow/meson.build:L405 .